src/Workspaces/Core/Portable/Classification/ClassificationExtensions.cs

```diff
@@ -9,6 +9,7 @@ internal static class ClassificationExtensions
         public static string? GetClassification(this ITypeSymbol type)
             => type.TypeKind switch
             {
+                TypeKind.Class when type.IsRecord => ClassificationTypeNames.RecordName,
                 TypeKind.Class => ClassificationTypeNames.ClassName,
```

Comment:
```suggestion
                TypeKind.Class => type.IsRecord ? ClassificationTypeNames.RecordName : ClassificationTypeNames.ClassName,
```

this was this is preserved as a jump table, instead of a series of if/else checks.